README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
-  <a href="https://www.notionjs.com/">
-    <img alt="Notion" src="https://github.com/notion-cli/notion/blob/master/notion.png?raw=true" width="480">
+  <a href="https://www.volta.sh/">
+    <img alt="Volta" src="./volta.png?raw=true" width="480">
```

Comment:
Can we also update the tag line below this to say "The JavaScript Launcher ⚡️" instead of "The Hassle-Free JavaScript Toolchain Manager"?

Reply:
GAH. I thought I had caught all of those! Thank you!

Reply:
Resolved in 529634ce.